balboa-client-kafka/README.md

```diff
@@ -0,0 +1,92 @@
+# Balboa Kafka Client
+
+This project provides an extensible library for publishing any serializable entity to an existing Kafka Cluster.
+
+## Motivation
+
+Kafka has become a proven distributed messaging bus for real-time data pipelines.  This library provides a mechanism for
+publishing messages to Kafka Clusters.  Currently, this project focuses heavily on publishing Balboa Metrics to an existing
+ cluster.
+
+## Installation
+
+This project is published as a maven repository and can be utilized and referenced as such through socrata's public
+ release library.
+
+SBT:
+1. Add the Socrata release repository to the project's `build.sbt`:
+
+```
+ resolvers := Seq(
+   "socrata maven" at "https://repository-socrata-oss.forge.cloudbees.com/release"
+ )
+ ```
+
+ 2. Add the balboa-client dependency to the project's library dependencies:
+
+ ```
+ libraryDependencies ++= Seq(
+   ...
+   "com.socrata" % "balboa-kafka-client" %% "0.14.0",
+   ...
+ )
+```
+
+Others: TODO
+
+## Usage and Code Example
+
+We are currently not allowing the auto creation of topic outside of testing environments to prevent unwanted namespace
```

Comment:
Let's move to auto create as discussed, it will make integration significantly easier for other teams, and I don't consider namespace conflicts to be a huge concern (and it would not solve that problem anyways)